README.md

```diff
@@ -135,6 +135,20 @@ git, I worked on github and gitlab. I tried using git guis but I prefer to work
 
 * Explain event delegation
 * Explain how `this` works in JavaScript
+    This points to an object depending on the context,
+    In what function was used in which we defined it.
+    Context, ie the way and place of the call.
+
+    Global context:
+    console.log(this === window); // true
+    this.a = 37;
+    console.log(window.a); // 37
+    As an object method:
+    var o = {  prop: 37, f: function() { return this.prop; }};
+    console.log(o.f()); // logs 37
+
+    https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Operatory/this
```

Comment:
możesz dodać też link
http://stackoverflow.com/questions/3127429/how-does-the-this-keyword-work